gnbsim.go

```diff
@@ -133,3 +132,30 @@ func getCliFlags() []cli.Flag {
 		},
 	}
 }
+
+func ListenAndLogSummary() {
+	for intfcMsg := range profctx.SummaryChan {
+		if intfcMsg.GetEventType() == common.QUIT_EVENT {
+			return
```

Comment:
Its like even if one profile QUITs, you will return ?

Reply:
No it will not return.
Profile routines never send QUIT signal on the SummaryChan, they only send their summary. Thus it will not return until it receives a QUIT signal which can be sent from the main routine, to stop the ListenAndLogSummary routine. Currently we do not send it as we expect this routine to run throughout the application lifecycle.